xds/src/main/java/io/grpc/xds/internal/security/certprovider/FileWatcherCertificateProviderProvider.java

```diff
@@ -38,6 +38,7 @@ final class FileWatcherCertificateProviderProvider implements CertificateProvide
   private static final String CERT_FILE_KEY = "certificate_file";
   private static final String KEY_FILE_KEY = "private_key_file";
   private static final String ROOT_FILE_KEY = "ca_certificate_file";
+  private static final String SPIFFE_FILE_KEY = "spiffe_trust_bundle_map_file";
```

Comment:
s/spiffe_trust_bundle_map_file/spiffe_trust_map_file

Reply:
This one comes from the spec, so we can't change it.